docs/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Installing Rancher Prime on AWS
+---
+
+## Installing Rancher  
+
+Log *helm* into the AWS Marketplace ECR to fetch the application. The AWS Marketplace ECR is always in the `us-east-1` region:
+
+```
+export HELM_EXPERIMENTAL_OCI=1
+
+aws --region us-east-1 ecr get-login-password \
+  | helm registry login --username AWS \
+  --password-stdin 709825985650.dkr.ecr.us-east-1.amazonaws.com
+```
+
+Install Rancher into your cluster using *helm*. Customize your helm installation values if needed:
+
+**NOTE**
+
+Rancher Prime utilizes cert-manager to issue and maintain its certificates. Rancher will generate a CA certificate of its own, and sign a cert using that CA.
+
+The Rancher hostname must be resolvable by public DNS. Please refer to the [Prerequisites](#prerequisites) section for more details.
+ 
+For example, if the DNS name is rancher.my.org,  HOST_NAME=rancher.my.org
+
+```shell
+helm install -n cattle-rancher-csp-deployer-system rancher-cloud --create-namespace \
+oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/{{repository}}/rancher-cloud-helm/rancher-cloud \
```

Comment:
```suggestion
oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/$REPOSITORY/rancher-cloud-helm/rancher-cloud \
```

Placeholder/variable syntax convention should be consistent, especially within the same command.